server/api/admin.ts

```diff
@@ -55,13 +55,15 @@ const getAuthorizationUrl = async (
   Logger.info(`code verifier  ${codeVerifier}`)
   // adding some data to state
   const newState = JSON.stringify({ app, random: state })
-  const url: URL = await google.createAuthorizationURL(newState, codeVerifier, {
-    scopes: oauthScopes,
-  })
-  // for google refresh token
-  if (app === Apps.GoogleDrive) {
-    url.searchParams.set("access_type", "offline")
-  }
+
+  const url: URL = google.createAuthorizationURL(
+    newState,
+    codeVerifier,
+    oauthScopes,
+  )
+  url.searchParams.set("access_type", "offline")
```

Comment:
add comment regarding the refresh token also retain the if condition that it is a Google App